Bluepill-cli/Bluepill-cli/Simulator/SimulatorScreenshotService.m

```diff
@@ -0,0 +1,159 @@
+//
+//  SimulatorScreenshotService.m
```

Comment:
please update the headers to be the same as other files (license info), same for other new files